app/views/partials/personal_key/_key.slim

```diff
@@ -7,7 +7,7 @@
     = t('users.recovery_code.header')
   .navy.monospace
     - code.split(' ').each do |word|
-      p.bold.mb0.h3 data-recovery='word'
+      p.bold.mb0.h3(data-recovery="#{word}")
```

Comment:
Holy moly! Since we know it's plain ASCII, we could just do `.innerHTML` and `.trim()` it to remove whitespace?